npm/css-src/sass/_success_summary.scss

```diff
@@ -0,0 +1,105 @@
+.success-summary {
+  border: 5px solid #00703c;
```

Comment:
Should this be using the design system's colour palette? Did you copy this code from the CSS or the Sass from the design system? I'd have expected the Sass to use colour variables.

Reply:
I copied the css. It didn't occur to me there'd be sass I could copy. I've copied the sass for the error summary component and changed the failure colours to success colours and I think that's better now.